internal/network/command_dns_forwarder_create.go

```diff
@@ -22,11 +27,15 @@ func (c *command) newDnsForwarderCreateCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "Create a DNS forwarder.",
-				Code: "confluent network dns forwarder create --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --domains abc.com,def.com ",
+				Code: "confluent network dns forwarder create --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --domains abc.com,def.com",
 			},
 			examples.Example{
 				Text: "Create a named DNS forwarder.",
-				Code: "confluent network dns forwarder create my-dns-forwarder --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --domains abc.com,def.com ",
+				Code: "confluent network dns forwarder create my-dns-forwarder --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456 --domains abc.com,def.com",
+			},
+			examples.Example{
+				Text: "Create a named DNS forwarder using domain-mapping. This will input a file which contains the domain mappings.",
```

Comment:
One possible alternative, it could be better:

```suggestion
				Text: "Create a named DNS forwarder using domain-mapping. This option reads the list of {domain, {zone, service}} mapping from a local file.",
```